demos/storybook/stories/user-menu/within-a-toolbar.tsx

```diff
@@ -16,7 +17,7 @@ export const withinToolbar = (): StoryFnReactReturnType => {
         title: {
             color: Colors.gray[600],
             fontSize: 12,
-            textAlign: 'right',
+            textAlign: getDirection() === 'rtl' ? 'left' : 'right',
```

Comment:
Is this necessary? Shouldn't the ILI be handling RTL correctly already?

Reply:
Yes, it is necessary. This gets rendered as an inline style. And we want to have a right aligned row

Reply:
Oh I see now, it's reversed (we want right aligned for LTR language)...got it.